### PR TITLE
Add a note in regards to issues when installing pre-commit

### PR DESCRIPTION
**Pull request by akaszynski**
Add a note in regards to installing `pre-commit` this [comment](https://github.com/pyvista/pyvista/pull/2708#issuecomment-1139951209).

### CODE REVIEW

**Review by tkoyama010 — APPROVED** [review]
LGTM

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -286,6 +286,10 @@ To ensure your code meets minimum code styling standards, run::
   pip install pre-commit
   pre-commit run --all-files
 
+If you have issues related to `setuptools` when installing `pre-commit`, see 
+[pre-commit Issue #2178 comment](https://github.com/pre-commit/pre-commit/issues/2178#issuecomment-1002163763)
+for a potential resolution.
```
Comment (adeak) [inline CONTRIBUTING.rst:291]: This needs to be ReSTified instead of markdownified, I'm afraid :)

**Review by adeak — CHANGES_REQUESTED** [review]
Formally requesting changes to make sure this isn't merged yet.

File: CONTRIBUTING.rst (hunk 1 above)
Comment (akaszynski) [inline CONTRIBUTING.rst:291]:
```suggestion
If you have issues related to ``setuptools`` when installing ``pre-commit``, see 
`pre-commit Issue #2178 comment <https://github.com/pre-commit/pre-commit/issues/2178#issuecomment-1002163763>`_
for a potential resolution.
```

Comment (akaszynski) [reply CONTRIBUTING.rst]: My bad. Fixed as requested in https://github.com/pyvista/pyvista/pull/2709/commits/14db08f6b8b31d54efa63d2b68f7de427c34d894.

**Review by adeak — APPROVED** [review]
LGTM now, thanks.

**Pull request merged**